openedx/features/wikimedia_features/admin_dashboard/course_reports.py

```diff
@@ -0,0 +1,35 @@
+# Create your views here.
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.csrf import ensure_csrf_cookie
+
+from common.djangoapps.edxmako.shortcuts import render_to_response
+from common.djangoapps.util.cache import cache_if_anonymous
+from lms.djangoapps.courseware.access import has_access
+from lms.djangoapps.courseware.courses import get_course_by_id, get_courses
+from lms.djangoapps.instructor.views.instructor_dashboard import _section_data_download
+
+
+@login_required
+@ensure_csrf_cookie
+@cache_if_anonymous()
+def course_reports(request):
+    courses_list = []
```

Comment:
Not commenting anything on this function as I assume you are using it just for testing purposes.